scripts/start-finalExec

```diff
@@ -110,9 +110,9 @@ if [[ ${GUI,,} = false ]]; then
   EXTRA_ARGS+=" nogui"
 fi
 
-: "${MEMORY:=1G}"
-: "${INIT_MEMORY:=${MEMORY}}"
-: "${MAX_MEMORY:=${MEMORY}}"
+: "${MEMORY=1G}"
```

Comment:
That looks like a syntax error, but I'll let the tests help determine that.

Reply:
My apologies, I never knew the colon was an option since I have never seen it done that way over all my years of experience. 

I'm still uncertain why the change was needed. An initial issue report and/or more background about what you tried would have helped the review process.